WhaleSpotting/react-app/src/components/Profile.tsx

```diff
@@ -57,15 +79,32 @@ export function Profile(): JSX.Element {
                         <img className="profile-image" alt="Profile Image" src="https://picsum.photos/id/237/200" />
                     </div>
                     <div className="button-container">
-                        <Button 
-                            style={Style.primary} 
+                        <Button
+                            style={Style.primary}
                             text="Sightings"
-                            onClick={() => setFeedToggle("Sightings")}/>
-                        <Button 
-                            style={Style.primary} 
+                            onClick={() => setFeedToggle("Sightings")} />
+                        <Button
+                            style={Style.primary}
                             text="Approvals"
                             onClick={() => setFeedToggle("Approvals")}
-                            dataTestId="approval-toggle"/>
+                            dataTestId="approval-toggle"
+                            forAdmin={true}
+                            isAdmin={isUserAdmin}
```

Comment:
if we have a hidden flag then this will just be 
```suggestion
                            hidden={!isUserAdmin}
```